Utils/SortAListInDescendingOrder.java

```diff
@@ -0,0 +1,20 @@
+package Utils;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SortAListInDescendingOrder<T extends Comparable<T>> {
+    public void sortInDescendingOrder(List<T> listToSortInDescendingOrder) {
+        try {
+            List sortedListInDescendingOrder = listToSortInDescendingOrder.stream()
+                    .sorted(Comparator.reverseOrder())
```

Comment:
Because each string began with an Upper Case Letter, Lower Case Letter or a number, it is sorted depending on that character. Each character has a numeric code point, and it differs from a Lower Case to Upper Case Character even if it is the same letter, it has a different numeric code point, and it sorted by that point. In our case the list is sorted descending that's why first of all are displayed string that starts with a lower case letter, then upper case letter and at the end numbers. Lower case letter has a greater numeric code point than upper case letter.